termcat-ld/README.md

```diff
@@ -0,0 +1,3 @@
+TERMCAT-LD are experiments towards the conversion of the terminological glossaries in Terminologia Oberta from the TERMCAT centre into RDF.
+Contact: Patricia Martín Chozas (patricia.martin@upm.es) 
+Github user: pmchozas
```

Comment:
```suggestion
TERMCAT-LD are experiments towards the conversion of the terminological glossaries in Terminologia Oberta from the TERMCAT centre into RDF.

### Contact

Patricia Martín Chozas <patricia.martin@upm.es> <br>
Github: pmchozas
```

Reply:
@pmchozas please commit these changes if you agree with them